src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt

```diff
@@ -705,14 +705,14 @@ class LicenceServiceTest {
       topupSupervisionExpiryDate = LocalDate.of(2021, 10, 22),
       comFirstName = "Stephen",
       comLastName = "Mills",
```

Comment:
Should we remove the first/last names everywhere, and maybe drop the columns from the licence table?  Are they used anywhere now?

Reply:
They're not used. I think there is a task to clean up the DB schema in general

Reply:
Will put one in as `tech debt`